### ISSUE TITLE
Server names missing apostrophes

### ISSUE THREAD

**Issue by lichtscheu** [stevin05/CraftScan#48]
Server names with apostrophes are not correctly in greetings. The apostrophe (') is omitted, resulting in an incorrect server name.

So instead of example-Mal'Ganis the greeting displays and whispers example-MalGanis.

**Comment by stevin05** [comment]
Ugh, I think i added the apostrophe removal to get something working around connected realms, but don't remember what. The realm name is also in saved variables, so will have to ensure the fix for this doesn't orphan those entries and essentially wipe your config.

**Comment by lichtscheu** [comment]
Yeah already played around with this one, i also tried to just replace the server name for the output maybe that is an option to make sure config does not get any problems and also works for the case you dont remember? 

Also removed the ' from ShortenRealmName() which worked in my tests but it will add a new character to config so there is something needed that replaces those character names once.
